src/views/Viewer.vue

```diff
@@ -389,9 +391,19 @@ export default {
 			// override mimetype if existing alias
 			const mime = fileInfo.mime
 			this.currentFile = new File(fileInfo, mime, this.components[mime])
+			this.changeSidebar()
 			this.updatePreviousNext()
 		},
 
+		/**
+		 * Show sidebar if sidebar is already opened
+		 */
+		changeSidebar() {
+			if (this.sidebarFile !== '') {
```

Comment:
Eheh, this doesn't check against null or undefined.
Meaning if the sidebar is not available (no OCA.Files.Sidebar) it will still try to open :)

I'll fix in a secondary pr